updatesnap/SnapVersionModule/snap_version_module.py

```diff
@@ -62,6 +62,33 @@ def process_snap_version_data(upstreamversion, snap_name, version_schema, has_up
     return f"{upstreamversion}-{packagerelease}"
 
 
+def process_rock_version_data(upstream_version, previous_version, version_schema, has_update):
+    """ Returns processed rock version"""
+
+    match = re.match(version_schema, upstream_version)
+    if not match:
+        logging.warning("Version schema does not match with rock repository version")
+        return None
+    upstream_version = match.group(1).replace('_', '.')
+
+    def version_tuple(v):
+        return tuple(map(int, v.split('.')))
+
+    upstream_tuple = version_tuple(upstream_version)
```

Comment:
Ehm... Is this a problem in github, or the function is really still there?

Reply:
Sorry I forgot to remove that functions... Now fixed it